components/patient-access-rules/api/src/test/java/org/phenotips/data/permissions/internal/DefaultCollaboratorTest.java

```diff
@@ -167,6 +167,32 @@ public void equalsTest() throws ComponentLookupException
         Assert.assertFalse(c.equals("other"));
     }
 
+    /** Basic tests for {@link Collaborator#equals(Object)}. */
```

Comment:
Javadoc is not up to date. Either remove or make it more relevant.

Reply:
Fixed.